packages/credit-card-integration/src/CreditCardPaymentMethodComponent.test.tsx

```diff
@@ -2,21 +2,18 @@ import {
     CheckoutSelectors,
     CheckoutService,
     createCheckoutService,
+    PaymentMethod,
 } from '@bigcommerce/checkout-sdk';
-import { mount, ReactWrapper } from 'enzyme';
-import { EventEmitter } from 'events';
+import '@testing-library/jest-dom';
```

Comment:
Could you please check if this is necessary? In theory, it is duplicated as we have it here.
https://github.com/bigcommerce/checkout-js/blob/dc02c45ab6691859d423e14139dd8547414a6a17/jest-setup.ts#L1

Reply:
The rest looks good.

Reply:
Hmm somehow this is not picking up the jest-setup changes 😕 Let me look at configs.